README.md

```diff
@@ -28,7 +28,7 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
```

Comment:
![image](https://github.com/HunterX-7/Hello-react-front-end/assets/93039359/36e296ca-791f-4fbc-961a-cfda0be91231)

- [ ] It would be nice if you fix the link (in both frontend and backend readme) highlighted in the image above to make your readme descriptive and professional 😊